src/app/components/ui/Row/index.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import Box from '@material-ui/core/Box';
+import { withStyles } from '@material-ui/core/styles';
+import clsx from 'clsx';
+
+const styles = {
+  root:{
+    display: 'flex',
+    flexDirection: 'row'
+  },
+  centered:{
+    justifyContent: 'center'
+  }
+}
+
+const Row = ({
+  classes,
+  centered,
+  ...props
+}) => (
+  <Box className={clsx(classes.root, centered && classes.centered)} {...props}/>
```

Comment:
This one reads better like
```suggestion
  <Box
       {...props}
      className={clsx(classes.root, centered && classes.centered)}
  />
```

Reply:
Fixed